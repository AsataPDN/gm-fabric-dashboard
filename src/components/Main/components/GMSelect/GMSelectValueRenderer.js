import React from "react";
import PropTypes from "prop-types";

import SecondaryText from "components/SecondaryText";

export default function GMSelectValueRenderer({ title, val }) {
  return (
    <span>
      <span>{title} </span>
      <SecondaryText>{val.label}</SecondaryText>
    </span>
  );
}

GMSelectValueRenderer.propTypes = {
  title: PropTypes.string.isRequired,
  val: PropTypes.object.isRequired
};
