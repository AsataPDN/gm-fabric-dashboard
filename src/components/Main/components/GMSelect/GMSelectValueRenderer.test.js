import React from "react";
import { shallow } from "enzyme";
import GMSelectValueRenderer from "./GMSelectValueRenderer";

describe("GMSelectValueRenderer", () => {
  it("matches snapshot", () => {
    const aGMSelectValueRenderer = shallow(
      <GMSelectValueRenderer title="Yo" val={{ label: "Yo label" }} />
    );
    expect(aGMSelectValueRenderer).toMatchSnapshot();
  });
});
