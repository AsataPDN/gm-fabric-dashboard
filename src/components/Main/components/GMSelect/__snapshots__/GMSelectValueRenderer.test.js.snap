// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GMSelectValueRenderer matches snapshot 1`] = `
<span>
  <span>
    Yo
     
  </span>
  <styled.span>
    Yo label
  </styled.span>
</span>
`;
